dependencies.yml

```diff
@@ -137,10 +140,10 @@ com.google.protobuf:
   protoc: { version: *PROTOBUF_VERSION }
 
 com.graphql-java:
-  graphql-java: { version: '18.0' }
+  graphql-java: { version: '18.2' }
```

Comment:
Missed in PR description. It is important because the dependencies section is automatically generated from the PR description. 😉

Reply:
Oops, thanks. 😉